### PR TITLE
[Manta] Add: cla assistant workflow to sign community PRs  

### PR DESCRIPTION
**Pull request by rishabhfsd**
## Description

closes:- #362 
- This PR has introduced the functionality of CLA assistant. This is basically a workflow (github action) that will trigger when there will be any contribution from community. 
- This workflow will ask contributor to sign CLA before merging a branch.
- Whenever a contributor opens a new pull request on  repository, assistant will check to see if they have a CLA on file. 
- If not, it will leave a comment on the pull request with instructions on how to sign one.
-  Once they’ve signed, it will update the pull request to note that it is now okay to accept.
---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Wrote unit tests.
- [ ] Updated relevant documentation in the code.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [ ] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
- [ ] Verify benchmarks & weights have been updated for any modified runtime logics
- [ ] If needed, bump `version` for every crate.
- [ ] If import a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- [ ] If needed, update our Javascript/Typescript APIs. These APIs are offcially used by exchanges or community developers.
- [ ] If we're going to issue a new release, freeze the code one week early(it depends, but usually it's one week), ensure we have enough time for related testing.

### CODE REVIEW

**Comment by rishabhfsd** [comment]
Two things are important 
1] PERSONAL_ACCESS_TOKEN supposed to be created and stored in secrets at repository level not globally.
2] To not to invoke cla workflow by Manta-Member's PR merge requests , one's username supposed be added in workflow's allow list. (currently I have added known)

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
```
Comment (ghzlatarev) [inline .github/workflows/cla.yml:3]: Let's get rid of the warnings.

File: .github/workflows/cla.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: |
+           'https://github.com/cla-assistant/github-action/blob/master/SAPCLA.md'
+          branch: 'manta'
+          allowlist: |
+            stechu,grenade,Dengjianping,Garandor,ghzlatarev,rishabhfsd,bot*
+
+ #below are the optional inputs - If the optional inputs are not given,
```
Comment (ghzlatarev) [inline .github/workflows/cla.yml:30]: Maybe just leave a link to the github actions page, we can read these comments there.

File: .github/workflows/cla.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+           I hereby sign the CLA') ||
+           github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: |
+           'https://github.com/cla-assistant/github-action/blob/master/SAPCLA.md'
```
Comment (ghzlatarev) [inline .github/workflows/cla.yml:25]: We shouldn't use this.
Take a look at Parity's https://cla.parity.io/sign_cla/20213174/3

Comment (rishabhfsd) [reply .github/workflows/cla.yml:25]: Sure. I will check with Parity's CLA.

**Pull request closed without merging**